packages/client/hmi-client/src/components/model-template/tera-model-template.vue

```diff
@@ -1,16 +1,25 @@
 <template>
-	<tera-infinite-canvas>
+	<tera-infinite-canvas
+		@save-transform="saveTransform"
+		@mouseenter="setMouseOverCanvas(true)"
+		@mouseleave="setMouseOverCanvas(false)"
+		@drop="onDrop"
+		@dragover.prevent
+		@dragenter.prevent
+		@focus="() => {}"
+		@blur="() => {}"
+	>
 		<template #foreground>
 			<aside>
 				<section v-if="model?.header?.schema_name">
-					<h7>Model framework</h7>
+					<header>Model framework</header>
 					<h5>{{ model.header.schema_name }}<i class="pi pi-info-circle"></i></h5>
 				</section>
 				<section>
-					<h7>Model templates</h7>
+					<header>Model templates</header>
 					<ul>
 						<li v-for="(_, index) in 5" :key="index">
-							<tera-model-template-card />
+							<tera-model-template-card :card="newCard" draggable="true" />
```

Comment:
Note: `#foreground` is meant as an overlay for the infinite canvas, you could put things like messages/status etc etc. I am not entirely sure if it is the best place to put a side panel. Having said that I don't really have any objections either.

Reply:
in the workflows the top toolbar is placed in `#foreground` so I saw this as a toolbar on the side.